Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi9/go-toolset:1.23.6-1744294473 AS builder
+FROM registry.access.redhat.com/ubi9/go-toolset:1.23.9-1751538372 AS builder
```

Comment:
Should this be 9-2 as well?

Reply:
@justinorringer no - this tag is more up to date, and is targeting ubi9 (not ubi8 like in the tekton files).